src/math/trigonometry.js

```diff
@@ -281,9 +281,10 @@ p5.prototype.radians = angle => angle * constants.DEG_TO_RAD;
 /**
  * Sets the current mode of p5 to the given mode. Default mode is RADIANS.
  *
+ * Calling <a href="#/p5/angleMode">angleMode()</a> with no arguments returns current anglemode.
  * @method angleMode
  * @param {Constant} mode either RADIANS or DEGREES
- *
+ * @chainable
```

Comment:
Why is this function made chainable? For most p5.js API there isn't much sense for functions to be chainable as they usually are not exposed as object instance methods.

Reply:
I really don't know the use of chaining in this type of functions i just used other mode functions as a reference: 

https://github.com/processing/p5.js/blob/e32b45367baad694b1f4eeec0586b910bfcf0724/src/core/shape/attributes.js#L139-L141
https://github.com/processing/p5.js/blob/e32b45367baad694b1f4eeec0586b910bfcf0724/src/core/shape/attributes.js#L36-L38
https://github.com/processing/p5.js/blob/e32b45367baad694b1f4eeec0586b910bfcf0724/src/color/setting.js#L241-L246

Reply:
Can `@chainable` be removed here? Thanks.

Reply:
I think it's done